blog/2020-07-18-temporal-transparency-update-2.md

```diff
@@ -31,11 +31,11 @@ Hope everyone is having a great Friday. I'm happy to share our second V1 status
 
 We're continuing to dedicate the majority of our time and energy to V1 code completion. Here are the items which we handled since the last update:
 
-![](/cms/dsadsd.png)
+![Completed items this week: 'Go SDK bench core framework changes'; 'Implement Go SDK bench test which reacts on signals'; 'Production build sometimes shows blank page (bug)'; 'Workflows page status dropdown not filtering workflows properly (github #57)'; 'TaskQueues Sharded Correctly'; 'Review github issues and create more stories if needed'; 'Remove EVENT_TYPE_REQUEST_CANCEL_EXTERNAL_WORKFLOW_EXECUTION_FAILED'; 'Rename DecisionTask and Decision'; 'Cassandra Consistency and SerialConsistency should be configurable'; 'CLI unable to show describe workflow output when it has heartbeat data'](/cms/dsadsd.png)
```

Comment:
Since this post is trying to use the image to convey specific names of completed items, it makes sense to explain them for folks who can't or won't read the image.

https://dequeuniversity.com/rules/axe/3.5/image-alt